### PR TITLE
[vue][745, 746, 742, 743, 739]- made all API calls a-sync and tooltip and alignment fix

### PR DESCRIPTION
**Pull request by ann-aot**
# Issue Tracking

JIRA: 
1. https://aottech.atlassian.net/browse/FWF-745 
2. https://aottech.atlassian.net/browse/FWF-746
3. https://aottech.atlassian.net/browse/FWF-742 
4. https://aottech.atlassian.net/browse/FWF-743
5. https://aottech.atlassian.net/browse/FWF-739 

# Changes
What are the main changes in PR?
1. made all API calls async
2. tooltip and alignment fix

### CODE REVIEW

File: camunda-formio-tasklist-vue/src/components/TaskList.vue [hunk 1]
```diff
@@ -418,23 +423,23 @@ export default class Tasklist extends Mixins(TaskListMixin) {
     this.userName = getUserName();
   }
 
-  timedifference (date: Date) {
+  timedifference(date: Date) {
```
Comment (kurianbenoy-aot) [inline camunda-formio-tasklist-vue/src/components/TaskList.vue:426]: violates eslint rules `space-before-function-paren`. All function changes are violating this rule, which will be shown as a warning.

Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/components/TaskList.vue:426]: After using prettier, again use eslint to fix this issue

File: camunda-formio-tasklist-vue/src/components/TaskList.vue [hunk 2]
```diff
@@ -25,29 +25,33 @@
...
       <b-col v-if="getFormsFlowTaskId && task" :lg="maximize ? 9 : 12" md="12">
-        <ExpandContract />
         <div class="cft-service-task-details">
-          <b-row
-            class="ml-0 cft-task-header task-header-title"
-            v-b-tooltip.hover
-            title="Task Name"
-          >
-            {{ task.name }}
+          <b-row>
```
Comment (kurianbenoy-aot) [inline camunda-formio-tasklist-vue/src/components/TaskList.vue:29]: The text for TaskName, ProcessName and applicationId doesn't look alligned. Would it be an issue to the client?

image

File: camunda-formio-tasklist-vue/src/components/TaskList.vue [hunk 3]
```diff
@@ -209,33 +212,34 @@
...
-                <div @click="onClaim">
+              <b-col
+                :class="task.assignee ? '' : 'd-flex align-items-end'"
+                v-else
+              >
+                <div @click="onClaim" v-b-tooltip.hover.left title="Claim task">
                   <span id="claimAssignee">
-                    <i class="fa fa-user" />
-                    Claim
+                    <i class="fa fa-user" />Claim
```
Comment (kurianbenoy-aot) [inline camunda-formio-tasklist-vue/src/components/TaskList.vue:240]: Would be good to give two spaces after <i> tag to give some gap after icon and Claim

File: camunda-formio-tasklist-vue/src/components/addons/ExpandContract.vue [hunk 4]
```diff
@@ -15,7 +25,7 @@ import TaskListMixin from "../mixins/TaskListMixin.vue";
 
 @Component
 export default class ExpandContract extends Mixins(TaskListMixin) {
-  clickMaxi () {
+  clickMaxi() {
```
Comment (kurianbenoy-aot) [inline camunda-formio-tasklist-vue/src/components/addons/ExpandContract.vue:28]: Eslint issue `space-before-function-paren`.

File: camunda-formio-tasklist-vue/src/services/camunda-rest.ts [hunk 5]
```diff
@@ -2,69 +2,75 @@ import { engine, formExt } from "../services/constants";
 import { bpmAxios } from "../services/axios";
 
 const CamundaRest = {
-  getProcessDefinitions (bearerToken: string, CamundaUrl: string) {
-    return bpmAxios(bearerToken, CamundaUrl).get(
+  async getProcessDefinitions(bearerToken: string, CamundaUrl: string) {
```
Comment (kurianbenoy-aot) [inline camunda-formio-tasklist-vue/src/services/camunda-rest.ts:5]: violates eslint rule to give a space before function definition

**Review by krishnan-aot — APPROVED** [review]

**Review by kurianbenoy-aot — COMMENTED** [review]
LGTM :)

Just found a mistyped message here in tasklist file

https://github.com/AOT-Technologies/forms-flow-ai-extensions/pull/61/files#diff-7a531c0a57d08e2ca56429bf893be5dc12557fbbb2ca3fe504190a8231d05887R45

**Pull request merged**